src/components/Postiterie.vue

```diff
@@ -146,16 +152,28 @@ export default {
     },
 
 
-
     // function resultToXML
     // TODO
     // return the XML from the result array[];
     resultToXML: function(){
-      let xml = "";
-    
+      let $this = this;
+      let xml = process.env.VUE_APP_XML_ENTETE;
+      //let xml = '<?xml version="1.0" encoding="UTF-8"?><mxGraphModel dx="1190" dy="757" grid="1" gridSize="10" guides="1" tooltips="1" connect="1" arrows="1" fold="1" page="1" pageScale="1" pageWidth="827" pageHeight="1169" math="0" shadow="0"><root><mxCell id="0" /><mxCell id="1" parent="0" />';
+      this.temps.forEach(function (el, index) {
+        xml+=$this.forXml(el, index)
+      })
+      xml+='</root></mxGraphModel>'
```

Comment:
This is still quite ugly. Why not using a template engine like `ejs` or any other

Reply:
Why ? we don't display the string

Reply:
`ejs` or any other templating engines just take variables with values, a template (be it HTML, XML, json, or anything) and executes the template giving it all the variables you set.

It doesn't have to be displayed, or to be plugged to `express` or to `slim` for it to give you back a string from a template and variables.

Reply:
Something around those lines should work
```
const ejs = require('ejs');
const template = require('./my_googledocs_template.xml');
const xml = ejs.render(template, {postits : this.temps});
// do whatever you like with that XML, display it, send it to gdocs, store it in a db ...
```

Reply:
vue js compiler say this : 

```
./src/components/docs_template.ejs 1:0
Module parse failed: Unexpected token (1:0)
You may need an appropriate loader to handle this file type.
> <?xml version="1.0" encoding="UTF-8"?><mxGraphModel dx="1190" dy="757" grid="1" gridSize="10" guides="1" tooltips="1" connect="1" arrows="1" fold="1" page="1" pageScale="1" pageWidth="827" pageHeight="1169" math="0" shadow="0"><root><mxCell id="0" /><mxCell id="1" parent="0" />
```

Reply:
* same with .xml extension

Reply:
Indeed, I think vue-cli only handles vue files, you probably need to extend its compiler to handle plain string imports with xml. Probably like you'd do with sass, or less or ...

A quick google search points to a solution : https://forum.vuejs.org/t/importing-text-file-as-string-in-vue-cli-3/37784

Reply:
We will do that after finishing the work to do this week